### ISSUE TITLE
Open-source?

### ISSUE THREAD

**Issue by STaRDoGG** [DockStation/dockstation#210]
@igor-lemon , Just curious, how come you don't upload the source here on GH so others can (potentially) contribute, and move the development along a lot faster? It's been close to half a year since the last release, and it seems to be a software that people are liking to use, but dev is movie at a crawl.

**Comment by cronopio** [comment]
Agreed, looks like is not open source, since sources are not available. Why is used an [Apache license](https://github.com/DockStation/dockstation/blob/master/LICENSE) if you are not uploading the sources?
Wanted to debug some issues I'm having but I found no way to debug

**Comment by ssbarnea** [comment]
Ouch, I am curious if this is even acceptable use of GitHub usage terms, apparently is use of github to host a closed source project.

**Comment by redtux** [comment]
I also just wanted to get the source as this app wants its users to login now, and I wanted to check what my credentials are really needed for. :) @igor-lemon, could you please comment on this and provide the respective sources? Thank you for this app and for your help! :+1:

**Comment by relrod** [comment]
+1, where is the source?

**Comment by luckydonald** [comment]
They just place the GitHub badge prominently on there website as they are using this as issue tracker. They seem to have no intention to OSS this, with is a pitty.
See #174.

**Comment by jamezrin** [comment]
I think @igor-lemon is keeping it closed source so that in the future he can make it more business oriented and make this a product that can generate a good outcome. I don't know if that's good or not but I still would love if this was open source. Maybe open sourcing it opens up new opportunities you wouldn't have with it being closed source, given the current trend of all things open source.

**Comment by STaRDoGG** [comment]
It definitely appears that keeping it closed source is holding back the software's progress.

**Comment by bristea** [comment]
FYI, there is always the https://github.com/contact/report-abuse which one could use to report it to github.

AFAIK that is breach of GitHub TOS and also a breach of Apache License which is also included with the repository.

**Comment by igor-lemon** [comment]
Hi guys! I wrote about it many times, the current DockStation is proprietary and we don't have plans to make it open source. The current GitHub repository just for support and user's issues.
I apologizeе for not answering for a long time.

**Comment by luckydonald** [comment]
@bristea which section?

**Comment by ssbarnea** [comment]
RIP DockStation

**Comment by Pasqualle** [comment]
@igor-lemon can you share your plan?

https://twitter.com/dock_station current status: 1 release every 2 years..

5 days ago in #274 you wrote that you work alone now, and just few hours per week. That is a big difference compared to 2016, when you quit your job to create DockStation.
https://medium.com/@dockstation/dockstation-and-why-we-created-it-6ecb800fe067

In 2017, #50 you wrote there will be a community and enterprise edition. Is it still the plan?

**Comment by igor-lemon** [comment]
@Pasqualle These plans are no longer relevant. I don't want to predict anything yet and talk about plans because according to the plans I should have already released the release more than a year ago. :D
The 95% of the release was ready for a long time, but I stumbled on one very complex feature (long syntax + short syntax with AST support).

**Comment by Pasqualle** [comment]
I should have written why I am asking about a plan.
If you do not have a business plan with DockStation any more, and you do not receive any income from it, why the source code is still not open? Why do you prefer to keep it proprietary?

**Comment by eshirvana** [comment]
> DockStation @dock_station·Sep 10, 2020
>Hi there! We haven't published anything for a long time. I just want to say that the project is not dead and we're still working on it. We'll publish a very large update soon.

https://twitter.com/dock_station/status/1304170378936422402?s=20

soon? when then?

**Comment by barakethan** [comment]
Looks like the original project was kitematic, isn't it?

**Comment by jscottelblein** [comment]
For any future readers, let's be honest; this project is abandonware, regardless of what the author keeps saying. The repo should be archived.

The only way to get a modern version of this app would be to clone it from scratch as best you can, and then build forward from there.

**Comment by eshirvana** [comment]
I moved away from this , I use free web- based open source app instead now:

[Portainer.io](https://github.com/portainer/portainer)

**Comment by Pasqualle** [comment]
https://github.com/DockStation/dockstation/issues/280#issuecomment-1090269898

As I understand the current project will not be open source. Igor is willing to create a new project, even as open source in case there is a strong sponsor.

**Comment by ssbarnea** [comment]
@Pasqualle Don't bother, this project is as good as dead, we have https://podman-desktop.io/ which is open-source.
